tests/components/participants/helpers/participantHelper.test.ts

```diff
@@ -0,0 +1,24 @@
+import '@testing-library/jest-dom';
+import { applyFilters } from '../../../../src/components/participants/helpers/participantHelper';
+
+import { mockParticipants } from './__fixtures__/participants';
+
+describe('ParticipantHelper_ApplyFilters', () => {
+  it('filters correctly the participant list', () => {
+    const searchText = 'msg systems ag';
+
+    // Apply filters based on the search text
+    const filteredParticipants = applyFilters(mockParticipants, searchText);
+
+    // Check if two participants match the search text
+    expect(filteredParticipants).toHaveLength(2);
+
+    // Check that the participant is the expected one (mockParticipants[0])
+    // Assuming "msg systems AG" is mockParticipants[0]
+    expect(filteredParticipants).toContain(mockParticipants[0]);
+
+    // Check that the participant is the expected one (mockParticipants[2])
+    // Assuming "msg systems ag" is mockParticipants[2]
+    expect(filteredParticipants).toContain(mockParticipants[2]);
```

Comment:
Please do not use indexed element as expected element. See similar comments above.

Reply:
If you are using`toContainEqual` then you do not have to specify the complete object its enough to specify the relevant properties in this case the property with the fitlered text `legalName`. 

```typescript
expect(filteredParticipants).toContainEqual({ legalName: 'msg systems AG' });
```

Advantages:
- It is more expressive. It shows that the list contains an object with a property that has the seached text. The other propes are just noise (irrelevant).
- If in the input test data (the list that is filtered) other than the tested porp is changed for some reason the test won't fail.